.gitmodules

```diff
@@ -1,5 +1,3 @@
-[submodule "integration_tests/oci-runtime-tests"]
+[submodule "tests/oci-runtime-tests/src/github.com/opencontainers/runtime-tools"]
 	path = tests/oci-runtime-tests/src/github.com/opencontainers/runtime-tools
 	url = https://github.com/opencontainers/runtime-tools.git
-	ignore = dirty
```

Comment:
I think we should still keep the `ignore = dirty` part ? Is it not needed?

Reply:
I have already addressed it.